src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -46,7 +47,7 @@ internal interface ITaintIssuesControlViewModel : INotifyPropertyChanged, IDispo
 
         ICommand ShowInBrowserCommand { get; }
 
-        ICommand ShowDocumentationCommand { get; }
+        INavigateToDocumentationCommand ShowDocumentationCommand { get; }
```

Comment:
Does the view model need to know that the command is an 'INavigateToDocumentationCommand` rather than just and `ICommand`?